packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -288,11 +253,24 @@ const ModalScrollContent = styled(Flex)<ModalScrollContentProps>`
   `};
 `
 
-const CloseIconWrapper = styled(Box)`
+// TODO: Consider extracting the equivalent of `TouchableWithoutFeedback`
+// TODO: Icon should support `currentColor` as a valid color
+const CloseButton = styled.button`
   position: fixed;
-  top: ${space(2)}px;
-  right: ${space(2)}px;
+  appearance: none;
+  top: 0;
+  right: 0;
   cursor: pointer;
+  padding: ${space(2)}px ${space(2)}px 0 0;
```

Comment:
Do we have a better way of doing this? (Other than attrs)

Reply:
This is how we've been doing it, outside of props / attrs.

Reply:
Why do we specify the spacing scale as numbers and not px literals if they are only ever px? Simpler to do math with?

Reply:
We could do a helper like https://github.com/ModaOperandi/tokens/blob/master/src/helpers.ts#L23 which would at least make this less verbose. I'd be interested to know if we can ascertain if we're in a string literal or not from inside the function then automatically append the `px`

Reply:
Would be lying if I said we _didn't_ use the `space` helper a good deal, but mostly we've used react props piped through to styled-system which gets around things.

The decision around the space scale came from the design team directly; we just copied their system: https://palette.artsy.net/tokens/spacing/

Reply:
Actually come to think of it; it'd be more proper to just use a `themeGet` from within so that you access the current theme. (So for instance, if we switch to a separate font/space scale for mobile, this method would prohibit us from swapping theme since the function is outside the context tree)

Reply:
Yeah ideally we'd be able to compose that somewhere [in here](https://github.com/artsy/palette/blob/master/packages/palette/src/helpers/space.ts#L7) so that a later refactor wouldn't be too drastic, but not sure how without direct access to context.